docs/pages/reference/resources.mdx

```diff
@@ -568,3 +569,76 @@ status:
 ```
 
 See [Teleport Agent Managed Updates](../upgrading/agent-managed-updates.mdx) for more details.
+
+## Access monitoring rule
+
+Access monitoring rules allows cluster administrators to monitor access requests
+and apply notification routing and automatic review rules.
+
+```yaml
+kind: access_monitoring_rule
+version: v1
+metadata:
+  name: access_monitoring_rule
```

Comment:
I think it would be helpful to come up with an example name which is not the same as the `kind` to help illustrate that these are separate values.